src/DataGridList/DataGridList.js

```diff
@@ -80,7 +80,7 @@ var DataGridList = module.exports = kind(
 	* @method
 	* @private
 	*/
-	constructor: kind.inherit(function (sup) {
+	create: kind.inherit(function (sup) {
```

Comment:
Might be a good idea to always return the result, though I don't think it matters for create.